packages/db/migrations/0105.undo.add_foreign_key_to_subscription.sql

```diff
@@ -0,0 +1,20 @@
+-- Type: UNDO
+-- Name: add_foreign_key_to_subscription
+-- Description: Add newsletter_email_id as foreign key to the subscription table
+
+BEGIN;
+
+-- remove old column
+ALTER TABLE omnivore.subscriptions
```

Comment:
Oh, I thought this one was already in there. Sorry, more work than I expected.

Reply:
Actually good catch so I can add the foreign key here!